### DISCUSSION TITLE
Touch devices - disable controls on first tap

### DISCUSSION THREAD

**Discussion by raze-sundaysky** [videojs/video.js#7455] [Q&A]
Hello,

When tapping on a device to display the controls, if the tap happens on an area the should have a control once displayed (for example the play/pause button), the control triggers.
Is there an option to disable the controls until the transition is complete?

Thanks!!

**Comment by mister-ben** [comment]
That doesn't happen with the standard controls e.g. at https://codepen.io/gkatsev/full/GwZegv Do you have customised controls?

**Reply by raze-sundaysky** [reply]
Thanks!
Yes, I will investigate further... 
Is there an event for when the control bar is shown? couldn't find one in the documentation...

**Reply by mister-ben** [reply]
Not exactly but there are ˋuseractiveˋ and ˋuserinactiveˋ events

**Reply by raze-sundaysky** [reply]
Thanks!
